css/reset.css

```diff
@@ -0,0 +1,45 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+ul,
+ol {
+  list-style: none;
+}
+
+a {
+  color: inherit;
+  text-decoration: none;
+}
+
+button {
+  color:inherit;
+  font-size: inherit;
+  background-color: inherit;
+  border: none;
+}
+
+select {
+  border: none;
+  appearance: none;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  font-weight: normal;
```

Comment:
It's better to use numeric values - 'font-weight: 400;'

Reply:
👌